src/components/migrationToV3.stories.mdx

```diff
@@ -83,6 +87,154 @@ const MyApp = () => {
     <button onClick={joinMeeting}>Join Meeting</button>;
   );
 };
+
+### Meeting Provider prop change
+Meeting provider no longer takes the following props:
```

Comment:
nit: better to have new lines surround the headings
Need to check other headings as well.
```
// new line here
# Heading 1
// new line here
## Heading 2
// new line here
Paragraph, ...
```